shared/mail/mail.go

```diff
@@ -315,6 +317,13 @@ func SendMail(c smtpClient, mail mailData, date time.Time) error {
 
 	if mail.messageID != "" {
 		headers["Message-ID"] = []string{mail.messageID}
+	} else {
+		templateString := "<%s@" + config.Hostname + ">"
+		randomStringLength := 16
+		randomString := model.NewRandomString(randomStringLength)
+		timestamp := fmt.Sprint(time.Now().Unix())
+		messageID := fmt.Sprintf(templateString, randomString+"-"+timestamp)
+		headers["Message-ID"] = []string{messageID}
```

Comment:
This is way too lengthy. It can be shortened to:

```go
msgID := fmt.Sprintf("<%s-%d@%s>", model.NewRandomString(16), time.Now().Unix(), config.Hostname)
headers["Message-ID"] = []string{msgID}
```

Reply:
I don't want 16 to look like a magic number here. I think one extra line will be fine